packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -30,6 +30,7 @@ export default class WalletService {
     this.create(
       {
         name: 'asw',
+        keystore: '{"master":{"privateKey":"","chainCode":""},"password":"0"}',
```

Comment:
why does the keystore have a password property?

Reply:
It`s for checking password for now.It will not be a property after updating the keystore structure.